corehq/apps/ota/views.py

```diff
@@ -139,21 +139,30 @@ def claim(request, domain):
     as_user_obj = CouchUser.get_by_username(as_user) if as_user else None
     restore_user = get_restore_user(domain, request.couch_user, as_user_obj)
 
-    case_id = unquote(request.POST.get('case_id', ''))
-    if not case_id:
+    case_ids = request.POST.getlist('case_id')
+
+    if not case_ids:
         return HttpResponse('A case_id is required', status=400)
 
     try:
-        if get_first_claim(domain, restore_user.user_id, case_id):
-            return HttpResponse(status=204)
+        case_ids_already_claimed = get_first_claims(domain, restore_user.user_id, case_ids)
+        if case_ids_already_claimed:
+            case_ids_to_claim = {case for case in case_ids if case not in case_ids_already_claimed}
```

Comment:
You can just keep this line and remove the rest of the if/else - if `case_ids_already_claimed` is empty, `case_ids_to_claim` will ends up identical to `case_ids`. The way you're doing it is indeed less work, but the savings is trivial, so it's worth making the code simpler.

Reply:
removed the if/else statements and yeah it still works as intended